cgi-bin/DW/Controller/Entry.pm

```diff
@@ -1038,7 +1038,7 @@ sub _do_post {
         my $extradata = {
             security => $form_req->{security},
             security_ml => "",
-            subject => LJ::ehtml( $form_req->{subject} ),
+            subject => $form_req->{subject},
```

Comment:
This is a bit misleading: the problem in #1453 is that the text there was passed as an argument to `form.textbox` and `form.hidden`, both of which escape their arguments when printing their form fields.  You're not using those below, so either this should be preserved, or (better) you should move the escaping into the TT view.  I'll point out what I think that should look like below.